### DISCUSSION TITLE
Naming - d2fdx2 vs df2dx2

### DISCUSSION THREAD

**Discussion by jaju** [ml-explore/mlx#1103] [General]
Hello!
In the following
https://github.com/ml-explore/mlx/blob/a9f80d60f6ade25efd20fd9fa715ecb33ffeed38/examples/cpp/tutorial.cpp#L92-L93

Should it not be d2fdx2 instead of df2dx2?

Regards,
jaju

PS: Appreciate the entire mlx effort!

**Comment by awni** [comment]
Yup, that looks like a typo! If you would be up for sending a PR to fix it, that would be great!

**Reply by jaju** [reply]
Thank you for your response and also the opportunity! Have sent a PR - https://github.com/ml-explore/mlx/pull/1104
